main.go

```diff
@@ -1,6 +1,113 @@
 package main
-import "fmt"
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"database/sql"
+	_ "github.com/lib/pq"
+)
+
+type server struct {
+	db *sql.DB
+}
+
+type Event struct {
+	Id        string `json:"id"`
+	Title     string `json:"title"`
+	Completed bool   `json:"completed"`
+}
+
+type Todo struct {
+	Id        string `json:"id"`
+	Title     string `json:"title"`
+	Completed bool   `json:"completed"`
+}
+
+type Person struct {
+	Id        string   `json:"id"`
+	Firstname string   `json:"firstname"`
+	Lastname  string   `json:"lastname"`
+	Location  string `json:"city"`
+}
+
+// $ export DBHOST=localhost
+// $ export DBPORT=5432
+// $ export DBUSER=you
+// $ export DBPASS=pass
+// $ export DBNAME=dbname
 
 func main() {
-    fmt.Println("hello world")
+	db, err := sql.Open(“postgres”,”user="DBUSER" dbname="DBNAME" sslmode=disable”)
+	if err != nil {
+		log.Fatal(err)
+	  }
+	defer db.Close()
+
+	events := append(events, Event{Id: "122", Title: "Las Vegas", Completed: true})
```

Comment:
`append()` joins two arrays together - you haven't made one here yet!

```go
events := make([]Event, 0)
events = append(events, Event{ /* ... */})
```

or even better:

```go
events := []Event{Event{}, Event{}, /*... */}
```

also note that just making them here doesn't do anything - you'll need to use that `db` object to insert this into the database